app/views/devise/confirmations/new.html.erb

```diff
@@ -3,7 +3,7 @@
 <%= form_for(resource, as: resource_name, url: confirmation_path(resource_name), html: { method: :post }) do |f| %>
   <%= render "devise/shared/error_messages", resource: resource %>
 
-  <div class="field">
+  <div class="">
```

Comment:
If you don't need to add classes, delete the class attribute

Reply:
fixed!!